packages/trpc/server/routers/viewer/dsync/create.handler.ts

```diff
@@ -0,0 +1,56 @@
+import type { DirectoryType } from "@boxyhq/saml-jackson";
+
+import jackson from "@calcom/features/ee/sso/lib/jackson";
+import { canAccess, samlProductID, samlTenantID, tenantPrefix } from "@calcom/features/ee/sso/lib/saml";
+import prisma from "@calcom/prisma";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { ZCreateInputSchema } from "./create.schema";
+
+type Options = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: ZCreateInputSchema;
+};
+
+// Create directory sync connection for a team
+export const createHandler = async ({ ctx, input }: Options) => {
+  const { orgId } = input;
```

Comment:
should we check here if orgId belongs to signed in user here?

Reply:
On line 25 as a part of `canAccess()` it checks if the user is an admin/owner of the org